vimeo-networking/src/main/java/com/vimeo/networking/model/TextTrack.java

```diff
@@ -0,0 +1,134 @@
+package com.vimeo.networking.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Created by Mohit Sarveiya on 6/29/17.
+ */
+@UseStag
+@SuppressWarnings({"unused", "WeakerAccess"})
+public class TextTrack implements Serializable {
```

Comment:
Needs a unit test to verify that the stag type adapter was generated for this class.